Sources/SwiftSyntax/SyntaxTreeViewMode.swift

```diff
@@ -27,7 +27,7 @@ public enum SyntaxTreeViewMode {
   /// Both missing and garbage nodes will be traversed.
   /// Mostly useful for SwiftSyntax internal verifiers that verify the integrity
   /// of the entire tree
```

Comment:
Comment still true?

Reply:
Good catch. I think we should get rid of it.